.docker/test.ini

```diff
@@ -108,6 +108,9 @@ ckan.views.default_views = image_view text_view recline_view
 # Customize which image formats the image_view plugin will show
 #ckan.preview.image_formats = png jpeg jpg gif
 
+## Front-End Settings
+ckan.favicon = /favicon.ico
```

Comment:
Is this the right favicon? As far as I can see, that URL doesn't exist. I think it should be `https://www.qld.gov.au/favicon.ico`

Reply:
Hi @ThrawnCA 
I've updated this now. That reference is for the Circle CI test ini file which we had added favicon (https://github.com/qld-gov-au/ckanext-data-qld-theme/blob/training-to-develop/ckanext/data_qld_theme/public/favicon.ico) prior.
Thanks.